msal/src/main/java/com/microsoft/identity/client/authorities/Authority.java

```diff
@@ -123,4 +126,9 @@ protected boolean getKnownToDeveloper() {
         return mKnownToDeveloper;
     }
 
+    public static String getAuthorityFromAccount(final IAccount account) {
+        return "https://login.microsoftonline.com/"
```

Comment:
This looks like a design problem that we need to resolve.  Don't we need to know which cloud the account is associated with?  Here we're using world wide as default?